docs/build/circuit-construction.ipynb

```diff
@@ -349,6 +403,105 @@
+      "text/plain": [
+       "<Figure size 203.683x367.889 with 1 Axes>"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc = QuantumCircuit(2, 2)\n",
+    "qc.measure(0, 1)\n",
+    "qc.draw(\"mpl\", cregbundle=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "79ba9f07",
+   "metadata": {},
+   "source": [
+    "* `QuantumCircuit.measure_all` : takes no argument and can be used for quantum circuits without pre-defined classical bits. It will create classical wires and store measurement results in order. It will also introduce a barrier before the measurement."
```

Comment:
also for this line in particular can you rephrase "store measurement results in order". its not clear to me here what you mean by "in order".